src/k8s-extension/setup.py

```diff
@@ -32,7 +32,7 @@
 # TODO: Add any additional SDK dependencies here
 DEPENDENCIES = []
 
-VERSION = "0.3.1"
+VERSION = "0.3.2"
```

Comment:
We have another version coming out 0.4.0 so depending on when this goes in, this version might need to be changed

Reply:
May I ask what is your expected version?

Reply:
This should now be `0.4.1`

Reply:
Yes, it has been updated